contracts/RMRK/extension/tokenProperties/IRMRKPropertiesAccessControl.sol

```diff
@@ -0,0 +1,157 @@
+     *  (`false`).
+     */
+    function registerAccessControl(
+        address collection,
+        address issuer,
+        bool useOwnable
+    ) external;
+
+    /**
+     * @notice Used to manage the access control settings for a specific parameter.
+     * @dev Only the `issuer` of the collection can call this function.
+     * @dev The `collaboratorAddresses` and `collaboratorAddressAccess` arrays must be of the same length.
+     * @dev The `specificAddresses` and `specificAddressAccess` arrays must be of the same length.
+     * @dev the possible `parameterType` values are:
+     *  [
+     *      STRING,
+     *      ADDRESS,
+     *      BYTES,
+     *      UINT,
+     *      BOOL
+     *  ]
+     * @dev The possible `accessType` values are:
+     *  [
+     *      Issuer,
+     *      Collaborator,
+     *      IssuerOrCollaborator,
+     *      TokenOwner,
+     *      SpecificAddress,
+     *  ]
+     * @dev Emits an {AccessControlUpdated} event.
+     * @param collection The address of the collection being managed.
+     * @param parameterType Type of the parameter being managed.
+     * @param parameterId ID of the parameter being managed.
+     * @param accessType The type of access control to be applied to the parameter.
+     * @param collaboratorAddresses The array of addresses to be added or removed from the list of collaborators.
+     * @param collaboratorAddressAccess The array of boolean values to indicate if the address should be added as a
+     *  collaborator (`true`) or removed (`false`).
+     * @param specificAddresses The array of addresses to be added or removed from the list of specific addresses.
+     * @param specificAddressAccess The array of boolean values to indicate if the address should be added as a specific
+     *  address (`true`) or removed (`false`).
+     */
+    function manageAccessControl(
+        address collection,
+        ParameterType parameterType,
+        uint256 parameterId,
+        AccessType accessType,
+        address[] memory collaboratorAddresses,
+        bool[] memory collaboratorAddressAccess,
+        address[] memory specificAddresses,
+        bool[] memory specificAddressAccess
+    ) external;
+
+    /**
+     * @notice Used to check if the specified address is listed as a collaborator of the given collection's parameter.
+     * @param collaborator Address to be checked.
+     * @param collection Address of the collection.
+     * @param parameterType Type of the parameter being checked.
+     * @param parameterId ID of the parameter being checked.
+     * @return Boolean value indicating if the address is a collaborator of the given collection's parameter (`true`) or
+     *  not (`false`).
+     */
+    function isCollaborator(
+        address collaborator,
+        address collection,
+        ParameterType parameterType,
+        uint256 parameterId
+    ) external view returns (bool);
```

Comment:
As stated previously:
1. I think collaborators should be independent from properties
2.I don't see the need for parameter type here, or anywhere in this contract. You shouldn't have multiple properties with different type and the same name, I would add it as a restriction.
3. Parameter Id is harder to use than simply a name

Reply:
We agree on the 1st point. 👌 
We agree on the 2nd point. 👌 
I still think we should keep it. The user never needs to pass the ID for anything else than management. We could expose a method to get ID for key, so that this process gets easier as well 🤔